action.sh

```diff
@@ -232,13 +242,13 @@ function start_vm {
 	./svc.sh install && \\
 	./svc.sh start && \\
 	gcloud compute instances add-labels ${VM_ID} --zone=${machine_zone} --labels=gh_ready=1
-	# we cancel any workflows longer than 5 hours (was originally 3 days, the max workflow runtime)
-	nohup sh -c \"sleep 5h && gcloud --quiet compute instances delete ${VM_ID} --zone=${machine_zone}\" > /dev/null &
+	# we cancel any workflows longer than 5 hours (was originally 3 days - the max workflow runtime)
```

Comment:
This comma was also confusing the gcloud command line tool that this entire thing is being passed to as a command-line argument. Would be nice to have this script live somewhere like in the image or at a URL which it can fetch rather than this interpolation/building.